src/lib/netlist/devices/nld_9316_base.hxx

```diff
@@ -8,10 +8,8 @@
 #include "nl_base.h"
 #include "nl_factory.h"
 
-namespace netlist
+namespace netlist::devices
 {
-	namespace devices
-	{
 
 		template <unsigned N>
```

Comment:
This is another file with an extra level of indentation now.

Reply:
Applied